service/src/test/resources/application.properties

```diff
@@ -1,6 +1,6 @@
-spring.datasource.url=
-spring.datasource.username=
-spring.datasource.password=
+spring.datasource.url=jdbc:mysql://localhost:3306/session
+spring.datasource.username=root
+spring.datasource.password=password
```

Comment:
I thought so too - it looked "disabled" in IntelliJ, which is why I didn't think anything about it.  I pulled in @millerts 's change in `develop` to resolve this issue.